src/libcalamaresui/ViewManager.cpp

```diff
@@ -183,8 +184,9 @@ ViewManager::onInstallationFailed( const QString& message, const QString& detail
     connect( msgBox, &QMessageBox::buttonClicked, [msgBox]( QAbstractButton* button ) {
         if ( msgBox->buttonRole( button ) == QMessageBox::ButtonRole::YesRole )
         {
-            // TODO: host and port should be configurable
-            QString pasteUrlMsg = CalamaresUtils::sendLogToPastebin( msgBox, QStringLiteral( "termbin.com" ), 9999 );
+            QString pasteURLHost = Calamares::Branding::instance()->logUploadURL();
+            int pasteURLPort = Calamares::Branding::instance()->logUploadPort();
+            QString pasteUrlMsg = CalamaresUtils::sendLogToPastebin( msgBox, pasteURLHost, pasteURLPort );
```

Comment:
naively, what is needed here is some sort of switch, eg:

```
QString pasteUrlMsg = (uploadServerType == 'fiche') ?
                      CalamaresUtils::sendLogToPastebin( msgBox, pasteURLHost, pasteURLPort ) :
                      QString();
```

passing in the hostname and port, to me is something of a smell - the current paste function is intrinsically specialized for a fiche server; so it could grab it's data directly from the branding class - besides, this logic really does not belong in a view class - (just an optional suggestion)

that would be sufficient to satisfy the feature request; but clearly not as DRY as it could be if, per my suggestion above, the paste function were an abstract sendLog() method of a subclass of an abstract UploadServer class:

```
QString pasteUrlMsg = uploadServer.sendLog( msgBox );
```
i would be leaning toward that approach; because it nicely avoids the need to modify the ViewManager class in the future